runelite-client/src/main/java/net/runelite/client/plugins/questlist/QuestListPlugin.java

```diff
@@ -324,9 +344,20 @@ private void updateList(QuestContainer questContainer, String filter)
 			Widget quest = questInfo.getQuest();
 			QuestState questState = QuestState.getByColor(quest.getTextColor());
 
-			boolean hidden = !(filter.isEmpty()
-				|| questInfo.getTitle().contains(filter))
-				|| (filter.isEmpty() && hideCompleted && questState == QuestState.COMPLETE);
+			if (questState == null) continue;
```

Comment:
Is this a bug in the existing plugin?

Reply:
While debugging I was getting NPEs on this variable later down the line that I could not really reliably reproduce. I am not sure what exactly could be the problem. I will try to find the root cause later today. I will update the PR if I find anything.